modules/testkit/src/test/scala/com/caesars/tracing/CompletedSpanQueue.scala

```diff
@@ -0,0 +1,22 @@
+package com.caesars.tracing
+
+import cats.effect.Ref
```

Comment:
Minor point: you might want to alias this to something like `CERef` to disambiguate between zio.Ref in general.